cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -383,6 +383,36 @@
           "widget-attributes" : {}
         }
       ]
+    },
+    {
+      "label": "Experimental Features (BETA)",
+      "properties": [
+        {
+          "name": "autoScaling",
+          "widget-type": "toggle",
+          "label": "Enable Autoscaling",
```

Comment:
Autoscaling

Reply:
Done. Changed Label from Enable Autoscaling to Autoscaling